src/ui/utils/matrixUtils.ts

```diff
@@ -0,0 +1,23 @@
+export const MX_PATH_PREFIX = "/_matrix/client/r0";
+
+export function getMxIdDomain(mxId: string) {
+  return mxId.slice(mxId.indexOf(":") + 1);
+}
+
+export async function isRoomAliasAvailable(homeserver: string, alias: string): Promise<boolean> {
```

Comment:
@bwindels do you think we could get a method in HomeServerApi to check for room alias availability? Or just a generic method to make requests with the proper headers set?